app/server/services/getAllQuestions.ts

```diff
@@ -7,6 +7,8 @@ async function getQuestion(hearingId) {
   try {
     const response = await request
       .get(`${apiUrl}/continuous-online-hearings/${hearingId}`);
+
+      console.log('TASK-LIST: get all questions', response.body)
```

Comment:
Is this just for debugging?

Reply:
yup, think I know the issue, but don't want to point fingers yet

Reply:
Nope, no need to point fingers except at the pipeline.  All passing now :angry: